streaming/src/main/scala/org/apache/spark/streaming/StreamingContext.scala

```diff
@@ -114,11 +123,15 @@ class StreamingContext private[streaming] (
 
   private[streaming] val isCheckpointPresent = (cp_ != null)
 
+  private[streaming] val isSparkContextPresent = (sc_ != null)
```

Comment:
Good point. My idea was to keep it consistent with isCheckpointPresent, which has the semantics of "was checkpoint present at the time of creation of the StremaingContext". So this was meant to be "was an existing sparkcontext used to create the stremaingContext". But i can see that it can be confusing. Will remove that completely.